src/components/file-upload/FileUpload.js

```diff
@@ -63,26 +63,28 @@ export default class FileUpload extends PureComponent {
     const {
       disabled,
       children,
-      dropZoneStyle,
       uploads,
       onNotify,
+      onReject,
+      onUpload,
```

Comment:
`onReject`, `onUpload` can be removed after https://github.com/ubergrape/grape-web-client/pull/722/files#r200601332